helm-charts/explorer/templates/explorer-configmap.yaml

```diff
@@ -0,0 +1,114 @@
+    app.kubernetes.io/name: {{ .Release.Name }}-app
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}  
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }} 
+data:
+
+  fabric-network.json: |-
+      {
+      "name": "fabric-network",
+      "version": "1.0.0",
+      "client": {
+        "tlsEnable": true,
+        "adminCredential": {
+          "id": "{{ .Values.explorer.username  | default "exploreradmin"}}",
+          "password": "{{ .Values.explorer.password | default "exploreradminpw"}}"
+        },
+        "enableAuthentication": true,
+        "organization": "{{ .Values.network.mspid}}",
+        "connection": {
+          "timeout": {
+            "peer": {
+              "endorser": "300"
+            },
+            "orderer": "300"
+          }
+        }
+      },
+      "channels": {
+        "{{ .Values.network.channel}}": {
+          "peers": {
+            "{{ .Values.network.peername}}": {}
+          },
+          "connection": {
+            "timeout": {
+              "peer": {
+                "endorser": "6000",
+                "eventHub": "6000",
+                "eventReg": "6000"
+              }
+            }
+          }
+        }
+      },
+      "organizations": {
+        "{{ .Values.network.mspid}}": {
+          "mspid": "{{ .Values.network.mspid}}",
+          "adminPrivateKey": {
+            "pem": "{{ .Values.network.adminPrivateKey}}"
+
+          },
+          "peers": [
+            "{{ .Values.network.peername}}"
+          ],
+          "signedCert": {
+            "pem": "{{ .Values.network.signedCert}}"
+          }
+        }
+      },
+      "peers": {
+        "{{ .Values.network.peername}}": {
+          "tlsCACerts": {
+            "pem": "{{ .Values.network.tlsCACerts}}"
+          },
+          "url": "{{ .Values.network.peerurl}}"
+        }
+      }
+      }
+
+  config.json: |- 
+      {
+      "network-configs": {
+        "fabric-network": {
+          "name": "fabric-network",
+          "profile": "./fabric-network.json"
+        }
+      },
+      "license": "Apache-2.0"
+      }
+
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-env
+  namespace: {{ $.Values.namespace }}
+  labels:
+    name: {{ .Release.Name }}-app
+    app.kubernetes.io/name: {{ .Release.Name }}-app
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }}     
+data:
+  DATABASE_HOST: {{ .Release.Name }}-db
+  DATABASE_DATABASE: {{ $.Values.db.database | default "fabricexplorer" }}
+  DATABASE_USERNAME: {{ $.Values.db.database_username | default "hppoc" }}
+  DATABASE_PASSWD: {{ $.Values.db.database_password | default "password" }}
```

Comment:
Similar to the private key, the password parameter is sensitive, it shall be loaded in from a secret store.